### PR TITLE
Remove repetitive logs printout

### PR DESCRIPTION
**Pull request by obino**

### CODE REVIEW

**Comment by obino** [comment]
ocd run https://ocd.appscale.com:8080/job/Daily%20Build/2676/

File: AppController/djinn.rb [hunk 1]
```diff
@@ -4939,7 +4938,11 @@ def check_running_apps()
         # need to do work only if we have AppServers.
         next unless info['appengine']
 
-        Djinn.log_debug("Checking #{app} with appengine #{info}.")
+        if info['appengine'].length > 10
```
Comment (menivaitsi) [inline AppController/djinn.rb]: Please make this a constant.

File: AppController/lib/haproxy.rb [hunk 2]
```diff
@@ -440,8 +439,16 @@ def self.list_servers(app, down=false)
         running << parsed_info[SERVICE_NAME_INDEX].sub(/^#{full_app_name}-/,'')
       end
     }
-    Djinn.log_debug("Haproxy: found these running AppServer for #{app}: #{running}.")
-    Djinn.log_debug("Haproxy: found these failed AppServer for #{app}: #{failed}.")
+    if running.length > 10
```
Comment (menivaitsi) [inline AppController/lib/haproxy.rb]: Same here.

**Comment by obino** [comment]
OCD https://ocd.appscale.com:8080/job/Daily%20Build/2682/

**Pull request merged**